src/libs/migrations/MoveReportHasDraftCollectionToReportObject.js

```diff
@@ -0,0 +1,68 @@
+import Onyx from 'react-native-onyx';
+import _ from 'underscore';
+import Log from '../Log';
+import ONYXKEYS from '../../ONYXKEYS';
+
+export default function () {
+    return new Promise((resolve) => {
```

Comment:
NAB, but not sure if the migration code is worth the complexity it adds. Is the worst case that we just start by assuming no reports have drafts and correctly track when a user edits the composer that has a draft in it? That seems pretty benign.

Reply:
^^ just confirm my understanding, this would wipe out all existing drafts on all reports, right?

Reply:
> Is the worst case that we just start by assuming no reports have drafts and correctly track when a user edits the composer that has a draft in it? That seems pretty benign.

Yeah, this is correct. It would also leave some stale data in Onyx that nothing is referencing. Also, not the end of the world. I am totally fine ditching the migration, but at least wanted to present what a true migration would be.

> this would wipe out all existing drafts on all reports, right?

If there was no migration, it doesn't delete the drafts, but it removes the edit/pencil icon in the LHN for any report with a draft.